### PR TITLE
add activity stream requestable for rewrite of notifications

### PR DESCRIPTION
**Pull request by bluwave**
refs: MBL-13525
affects: Student
release note: none

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1012049&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

File: Core/Core/ActivityStream/GetActivityRequestable.swift [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+import Foundation
+
+public struct GetActivityRequest: APIRequestable {
+    public typealias Response = APIActivity
```
Comment (vanwagonet) [inline Core/Core/ActivityStream/GetActivityRequestable.swift]: is this supposed to get an array of activities? If so, `GetActivitiesRequest` might be more suitable a name. Also you can just move this into the APIActivity.swift file

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: Student

[MBL-13525](https://instructure.atlassian.net/browse/MBL-13525)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.17% | 93.17% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Pull request merged**